zanata-war/src/main/java/org/zanata/dao/ApplicationConfigurationDAO.java

```diff
@@ -1,12 +1,14 @@
 package org.zanata.dao;
 
 import org.hibernate.Session;
+
+import javax.enterprise.context.RequestScoped;
 import javax.inject.Named;
 import org.zanata.model.HApplicationConfiguration;
 
 @Named("applicationConfigurationDAO")
 
-@javax.enterprise.context.Dependent
+@RequestScoped
```

Comment:
I think you're right... all other DAOs are request scoped. In light of this I remove my objection :wink: 

@seanf @huangp  Do you remember why we had to make them Request scoped instead of dependent? Proxies perhaps?